core/src/text/mod.rs

```diff
@@ -95,22 +85,22 @@ pub struct TextMetrics {
     pub next_position: Point,
 }
 
-/// Vertical text alignment.
+/// Text baseline.
 #[derive(Debug, Copy, Clone, Ord, PartialOrd, Eq, PartialEq, Hash)]
-pub enum VerticalAlignment {
+pub enum Baseline {
     /// Top.
     Top,
     /// Bottom.
     Bottom,
-    /// Center.
-    Center,
-    /// Baseline.
-    Baseline,
+    /// Middle.
+    Middle,
+    /// Alphabetic baseline.
```

Comment:
I think these could/should be explained better than just echoing the variant names.

For example I'm not entirely sure what `Alphabetic` refers to, although I have a guess. Also, does Top refer to the topmost pixel of the character, or something else? Perhaps we could adapt the text from the linked canvas docs.

Reply:
The docs need some work in general, I'll add a note to the tracking issue to revisit this later.

> For example I'm not entirely sure what Alphabetic refers to, although I have a guess.

The main reason I changed it from `Baseline` to `Alphabetic` is that `Baseline::Baseline` would look weird. But it also matches the canvas API. I don't think we will ever support ideographic or hanging baselines in the internal text renderer.

> Also, does Top refer to the topmost pixel of the character, or something else?

It refers to the top of the glyph bounding box. This can include additional blank pixels above the caps if the font includes them.

Reply:
I mean, I know what these are, but it's not obvious from the docs :)

Reply:
> I know what these are, ... ,but it's not obvious from the docs :)

The docs for the font/text stuff really need some work. Contributions to the docs are always welcome :wink: